version.sbt

```diff
@@ -1 +1 @@
-version in ThisBuild := "0.0.33-SNAPSHOT"
+version in ThisBuild := "0.0.34-SNAPSHOT"
```

Comment:
Isn't this handled by the Jenkins build now. It creates a new PR for the version bump?

Reply:
It is but I needed it deployed so the tests wouldn't fail on the other builds so I deployed it manually :grimacing:

Reply:
Tut, tut